src/Core/Domain/Security/ProviderConfiguration/OpenId/Model/OpenIdConfiguration.php

```diff
@@ -28,6 +28,13 @@
 class OpenIdConfiguration
 {
     public const DEFAULT_LOGIN_GLAIM = 'preferred_username';
+    public const TYPE = 'openid';
+    public const NAME = 'openid';
+
+    /**
+     * @var int|null
+     */
+    private ?int $id;
```

Comment:
Suggestion but not mandatory here : Create NewOpenIdConfiguration model which does not contain `id` property